views/species-index.ejs

```diff
@@ -0,0 +1,17 @@
+<%- include('partials/header', {title:'placeholder'}) -%>
+<div class="container">
+  <% for(let i =0; i<species.length; i+=1){ %>
```

Comment:
Since `{ species }` returns an array, you could use a forEach here  to simplify the EJS variables